upi/metal/terraform.tfvars.example

```diff
@@ -42,6 +42,15 @@ matchbox_trusted_ca_cert = "matchbox/tls/ca.crt"
 // The Project ID for Packet.net where servers will be deployed.
 packet_project_id = ""
 
+// The Packet.Net device plan slug.
+packet_plan = ""
```

Comment:
Set defaults for `packet_plan` and `packet_facility` here, as well.

Reply:
Fixed.